angular-spa-ui/js/search-app/search-module/advanced-search/advanced-search-ctrl.js

```diff
@@ -1 +1,6 @@
 "use strict";
+
+module.exports = function ($scope) {
+    console.log( $scope );
```

Comment:
You do not need to separate function argument with spaces, it should looks: console.log($scope) and double check do you really need this debug information in pull request.